app/Http/Middleware/PostOwner.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use App\Contracts\Services\PostServiceInterface;
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class PostOwner
+{
+    protected $postService;
+    function __construct(PostServiceInterface $postService)
+    {
+        $this->postService = $postService;
+    }
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next)
+    {
+
+        
```

Comment:
Delete empty line.
You must format your code file before you push it to git remote.